cmd/microservice-sui-worker/util.go

```diff
@@ -0,0 +1,54 @@
+package main
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/fluidity-money/fluidity-app/lib/log"
+	"github.com/fluidity-money/fluidity-app/lib/queue"
+	"github.com/fluidity-money/fluidity-app/lib/queues/worker"
+	"github.com/fluidity-money/fluidity-app/lib/types/applications"
+	worker_types "github.com/fluidity-money/fluidity-app/lib/types/worker"
+)
+
+const (
+	FluidityModule = `fluidity_coin`
```

Comment:
Make this an enviornment variable because of the multiple deployments we'll have to do